admin/simple-mechanics/wrapper.md

```diff
@@ -0,0 +1,9 @@
+---
+sidebar_position: 8
+---
+
+# Wrapper
+
+The main idea of the **Wrapper** mechanic is that it can **wrap an unlimited amount of [assets](/admin/miscellaneous/asset/)** into a single token. The **Wrap token** represents all of the [assets](/admin/miscellaneous/asset/) that were wrapped, and can be transferred to anyone in a single transfer. When a user wants to unwrap the Wrap token, they can call the "**unpack**" function and pass the **token ID** of the Wrap token. This will transfer all of the wrapped tokens back to the owner's address, and the *Wrap token* will be **burned**.
```

Comment:
remove "an unlimited amount of" it is technically limited by transaction gas limit